src/js/ResponsiveNav.js

```diff
@@ -156,6 +156,8 @@ function ResponsiveNav(rootEl) {
 		contentFilterEl = rootEl.querySelector('ul');
 		moreEl = rootEl.querySelector('[data-more]');
 
+		rootDelegate.on('oSquishyList.change', contentFilterChangeHandler);
```

Comment:
Because with the change to squishy list it stops the menu button displaying as the 'change' event is only triggered at instantiation not on every call to squish. Hope that helps.